client-src/default/index.js

```diff
@@ -136,7 +136,11 @@ const onSocketMessage = {
     log.error(error);
   },
   close() {
-    log.error('[WDS] Disconnected!');
+    if (options.hot || options.liveReload) {
+      log.error('[WDS] Disconnected!');
+    } else {
+      log.error('[WDS] Hot Module Replacement & Live Reloading are disabled!');
+    }
```

Comment:
I think it is unnecessary here to write `log.error('[WDS] Hot Module Replacement & Live Reloading are disabled!');`

/cc @Loonride @hiroppy

Reply:
Agree